sms_sponsorship/__manifest__.py

```diff
@@ -16,7 +16,10 @@
     'license': 'AGPL-3',
     'website': 'http://www.compassion.ch',
     'depends': ['crm_compassion', 'cms_form'],
-    'data': [],
+    'data': [
+        'data/sms_hold_release.xml',
+        'data/event_compassion_view.xml'
```

Comment:
views should be placed inside a `views` folder for consistency. You should add as well a view for displaying the new fields you added in the compassion.hold object.